modules/javafx.base/src/main/java/javafx/collections/FXCollections.java

```diff
@@ -318,14 +318,19 @@ public static ObservableFloatArray observableFloatArray(ObservableFloatArray arr
     }
 
     /**
-     * Creates a new empty observable list backed by an arraylist.
+     * Creates a new empty {@code ObservableList} backed by an array list that listens to changes in observables of its items.
+     * The {@code extractor} parameter specifies observables (usually properties) of the objects in the created list. When there is
+     * a change in one of those observables, the user is notified of it through an
+     * {@link ListChangeListener.Change#wasUpdated() update} change of an attached {@code ListChangeListener}. These changes
+     * are unrelated to the changes made to the observable list itself using methods such as {@code add} and {@code remove}.
+     * <p>
+     * For example, a list of {@code Shape}s can listen to changes in the shapes' {@code fill} property.
      *
-     * This list reports element updates.
      * @param <E> The type of List to be wrapped
-     * @param extractor element to Observable[] convertor. Observable objects are listened for changes on the element.
+     * @param extractor element to Observable[] converter. Observable objects are listened for changes on the element.
```

Comment:
You can probably drop the second sentence (and thus the period after the first) if you copy the following from `observableList` to the description of this method:

```
Observable objects returned by the extractor (applied to each list element) are listened for changes...
```